website/src/constants/guidebook.ts

```diff
@@ -0,0 +1,280 @@
+import {
+  Info,
+  Star,
+  MapPin,
+  MessageSquare,
+  Menu,
+  CheckSquare,
+  Truck,
+  Award,
+  Pill,
+  Calendar,
+  Map,
+  Users,
+  Wrench,
+  Upload,
+  Phone,
+  MessageCircle,
+  Utensils,
+  Building,
+  LucideIcon,
+} from "lucide-react";
+
+// Define icon names as strings instead of JSX elements
+export type IconName =
+  | "info"
+  | "star"
+  | "mapPin"
+  | "messageSquare"
+  | "menu"
+  | "checkSquare"
+  | "truck"
+  | "award"
+  | "pill"
+  | "calendar"
+  | "map"
+  | "users"
+  | "wrench"
+  | "upload"
+  | "phone"
+  | "messageCircle"
+  | "utensils"
+  | "building";
+
+// Map of icon names to their components
+export const iconMap: Record<IconName, LucideIcon> = {
+  info: Info,
+  star: Star,
+  mapPin: MapPin,
+  messageSquare: MessageSquare,
+  menu: Menu,
+  checkSquare: CheckSquare,
+  truck: Truck,
+  award: Award,
+  pill: Pill,
+  calendar: Calendar,
+  map: Map,
+  users: Users,
+  wrench: Wrench,
+  upload: Upload,
+  phone: Phone,
+  messageCircle: MessageCircle,
+  utensils: Utensils,
+  building: Building,
+};
+
+export interface GuidebookSubsection {
+  id: string;
+  title: string;
+  icon: IconName;
+  content: string;
+}
+
+export interface GuidebookSection {
+  id: string;
+  title: string;
+  icon: IconName;
+  subsections: GuidebookSubsection[];
+}
+
+export const GUIDEBOOK_DATA: GuidebookSection[] = [
+  {
+    id: "general",
+    title: "General Information",
+    icon: "info",
+    subsections: [
+      {
+        id: "checklist",
+        title: "Participant Checklist",
+        icon: "checkSquare",
+        content: `<h3 class="text-xl font-bold mb-1">🤝 Must Haves:</h3>
+• Valid ID
+• Laptop
+• Charger
+• Other Devices you may need
+• Water bottle
+• Comfortable clothes
+• Good Vibes :)
+
+<h3 class="text-xl font-bold mb-1">😴 If you plan to stay overnight:</h3>
+• Air mattress
+• Sleeping bag
+• Small Pillow
+• Sleeping mask
+• Ear plugs
+• ...
+
+<h3 class="text-xl font-bold mb-1">🔑 Essentials:</h3>
+• Basic toiletries: Toothpaste, Toothbrush, Shampoo, Deodorant, Medications, etc.
+• Towel (You will get free access to shower at the venue (ground floor))
+• ...`,
+      },
+      {
+        id: "transport",
+        title: "Transportation Options",
+        icon: "truck",
+        content: "Information about transportation to and from the event.",
```

Comment:
@nilsreichardt do you got any photos/videos from how to enter the building at Celonis?

Reply:
Yes, I will a Google Drive link ASAP